config/webpack-config.js

```diff
@@ -17,17 +17,27 @@ const COMMON_BUNDLE_NAME = 'common.js';
 const COMMON_MODULE_CONFIG = {
   loaders: [ {
     test: /\.js$/,
-    loaders: [ {
+    loaders: {
       loader: 'babel-loader?cacheDirectory=true',
       options: {
         presets: [ [ 'env', {
+          modules: false,
           targets: {
             browsers: BROWSER_LIST.LAST_2_IE_9_UP
           },
           debug: true
         } ] ]
       }
-    } ],
+    } ,
+    exclude: {
+      test: /node_modules/,
+      // The below regex will capture all node modules that start with `cf`
+      // or atomic-component. Regex test: https://regex101.com/r/zizz3V/1/.
+      exclude: /node_modules\/(?:cf.+|atomic-component)/
+    }
+  }, {
+    test: /\.js$/,
+    loader: 'webpack-rollup-loader',
```

Comment:
https://github.com/erikdesjardins/webpack-rollup-loader#usage shows webpack-rollup-loader appearing before babel. Here's it's after. Does that make a difference?

Reply:
That is a good question, I'll check on this.